### DISCUSSION TITLE
Automatic Identification DOI, Authors , Journals in Zotfile-MdNotes-Obsidian Work

### DISCUSSION THREAD

**Discussion by davidRFB** [BryanJenksCommunity/FAQ#121] [Q&A] [answered]
Hi Bryan ! Before start, I Just want to thank you. I saw your workflow of zotero and obsidian, and it is totally amazing !! 

I have a quick question . Is it possible to automatically extract the DOI, Authors names and Journal Name from ZotFile directly in order to export a markdown ( using MdNotes) and create easily the perfect note. 

An example : 
Imagine that I'm using zotero and take some notes and highlights. When I export the annotations, I would like to have something like : 
{DOI} 
{Authors}
{Journal Name}
---
ANNOTATIONS 
.
.
.
--

Then the markdown from MdNotes will have everything to just add tags and links in obsidian.
Maybe you know other possible solution . 

Thanks again 
David

**Comment by tallguyjenks** [comment, marked as answer]
some good options to look at in this collected forum post :)

- https://forum.obsidian.md/t/zotero-best-practices/164/57
